portfolio/src/main/webapp/index.html

```diff
@@ -64,10 +57,15 @@ <h2 class="hello-text">Hello,</h2>
           my GitHub.
         </p>
       </div>
+
+      <div id="step2">
+        <h2>Step 3: fetch JSON data from server and add to DOM (using body onload)</h2>
+        <div id="fetched-data"></div>
+      </div>
```

Comment:
should this be a comment? this looks like a TODO

Reply:
oh this is a demo of step 3's JSON string fetching